instaclustr/resource_cluster.go

```diff
@@ -725,6 +709,64 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error{
 	return nil
 }
 
+func getBundlesFromCluster(cluster *Cluster) ([]map[string]interface{}, error) {
+	baseBundle := make(map[string]interface{}, 0)
```

Comment:
**[Required, Memory Usage]** This map will always have 3 entries, since there are no loop based assignments. They are as follows:
```go
baseBundle["bundle"] = cluster.BundleType
baseBundle["options"] = convertedBundleOptions
baseBundle["version"] = cluster.BundleVersion
```